terminus-ui/chip/src/chip.component.spec.ts

```diff
@@ -17,61 +13,83 @@ import {
   dispatchFakeEvent,
 } from '@terminus/ngx-tools/testing';
 import * as testComponents from '@terminus/ui/chip/testing';
+
 import {
   TsChipComponent,
   TsChipModule,
   TsChipSelectionChange,
 } from './chip.module';
 
-describe('Chips', () => {
 
+describe('Chips', () => {
   let fixture: ComponentFixture<any>;
   let chipDebugElement: DebugElement;
   let chipNativeElement: HTMLElement;
   let chipInstance: TsChipComponent;
-  const globalRippleOptions: RippleGlobalOptions = { disabled: false };
 
   function createComponent<T>(component: Type<T>): ComponentFixture<T> {
     const moduleImports = [
       TsChipModule,
       NoopAnimationsModule,
     ];
-    const providers = [
-      {
-        provide: MAT_RIPPLE_GLOBAL_OPTIONS,
-        useFactory: () => globalRippleOptions,
-      },
-    ];
-    return createComponentInner(component, providers, moduleImports);
+    return createComponentInner(component, [], moduleImports);
   }
 
   describe('TsChip', () => {
-    let testComponent: testComponents.SingleChip;
-
-    beforeEach(() => {
-      fixture = createComponent(testComponents.SingleChip);
+    let testComponent;
+    function setup(component = testComponents.SingleChip) {
```

Comment:
The way this was set up originally required all the tests to use a single test host, so I needed to move to a setup function since I had a new test that needed a custom host.

Reply:
yep. at the time I didn't need a 2nd host component.. thus set up that way.